dev-docs/bidders/prebidServer.md

```diff
@@ -32,11 +32,14 @@ pbjs.setS2SConfig({
 Configuration options
 
 {: .table .table-bordered .table-striped }
-| Name | Required? | Description
-|:-----------|:-----------|:---------------------------|
-| `accountId` | required | string:required: account ID obtained in sign up process |
-| `enabled` | required | boolean:required: enables s2s - default false |
-| `bidders` | required | array[string]:required: of bidder codes to enable S2S. |
-| `syncEndpoint` | optional | string:optional sets user-sync endpoint. |
-| `timeout` | optional | number:optional timeout in ms for bidders called via the S2S endpoint.|
-| `cookieSet` | optional | boolean:optional: If `false` (not recommended), opt out of link rewriting to improve cookie syncing. |
+| Field        | Type          | Required? | Description                                                              |
```

Comment:
Can your editor auto format the table layout?  Not required, just a nice to have.

Each *Description* cell should end with a period (some do, some don't).